Auto update failing for procs

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the procs package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating procs [Current version: 0.13.1]
INFO: package being updated to 0.13.2.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 1057k    0 1057k    0     0  1322k      0 --:--:-- --:--:-- --:--:-- 1322k

You are about to commit these changes:

--------------------
diff --git a/packages/procs/build.sh b/packages/procs/build.sh
index e83fde842..87fb31995 100644
--- a/packages/procs/build.sh
+++ b/packages/procs/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/dalance/procs
 TERMUX_PKG_DESCRIPTION="A modern replacement for ps"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.13.1"
+TERMUX_PKG_VERSION="0.13.2"
 TERMUX_PKG_SRCURL=https://github.com/dalance/procs/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=29a5058f8389986e5b17117d3ee4f78a2bf9c8886e07b16a49a94e5fc59a6850
+TERMUX_PKG_SHA256=521b25cbb3fac8b6f3cfd33c4d58f269e7d6637a89f6b932f144a8bed7914ec0
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_AUTO_UPDATE=true
 
--------------------

procs: update to 0.13.2


Not committing to Git!

INFO: Trying to build package.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building procs for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0  71509      0 --:--:-- --:--:-- --:--:-- 71765
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2053      0 --:--:-- --:--:-- --:--:--  2056
gpg: Signature made Wed 05 Oct 2022 12:41:14 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 78 13319   78 10411    0     0  23941      0 --:--:-- --:--:-- --:--:-- 23933
100 13319  100 13319    0     0  30614      0 --:--:-- --:--:-- --:--:-- 30618
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2071      0 --:--:-- --:--:-- --:--:--  2072
gpg: Signature made Sun 04 Sep 2022 06:32:25 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0  42509      0 --:--:-- --:--:-- --:--:-- 42594
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2589      0 --:--:-- --:--:-- --:--:--  2586
gpg: Signature made Sun 02 Oct 2022 05:46:36 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://github.com/dalance/procs/archive/refs/tags/v0.13.2.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  145k    0  145k    0     0   219k      0 --:--:-- --:--:-- --:--:--  219k
100 1057k    0 1057k    0     0  1292k      0 --:--:-- --:--:-- --:--:-- 5847k
dirname: missing operand
Try 'dirname --help' for more information.
/home/builder/termux-packages/packages/libllvm/build.sh: line 63: [: =: unary operator expected
info: downloading installer
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
info: syncing channel updates for '1.64.0-x86_64-unknown-linux-gnu'
info: latest update on 2022-09-22, rust version 1.64.0 (a55dd71d5 2022-09-19)
info: downloading component 'cargo'
info: downloading component 'clippy'
info: downloading component 'rust-docs'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: downloading component 'rustfmt'
info: installing component 'cargo'
info: installing component 'clippy'
info: installing component 'rust-docs'
info: installing component 'rust-std'
info: installing component 'rustc'
info: installing component 'rustfmt'

info: default toolchain set to '1.64.0-x86_64-unknown-linux-gnu'
  1.64.0-x86_64-unknown-linux-gnu installed - rustc 1.64.0 (a55dd71d5 2022-09-19)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, run:
source "$HOME/.cargo/env"
info: downloading component 'rust-std' for 'aarch64-linux-android'
info: installing component 'rust-std' for 'aarch64-linux-android'
    Updating crates.io index
 Downloading crates ...
  Downloaded async-executor v1.4.1
  Downloaded fnv v1.0.7
  Downloaded kv-log-macro v1.0.7
  Downloaded byte-unit v4.0.14
  Downloaded dirs-sys v0.3.7
  Downloaded console v0.15.2
  Downloaded lazy_static v1.4.0
  Downloaded matches v0.1.9
  Downloaded mime v0.3.16
  Downloaded bitflags v1.3.2
  Downloaded proc-macro-error-attr v1.0.4
  Downloaded mio v0.8.4
  Downloaded once_cell v1.15.0
  Downloaded parking v2.0.0
  Downloaded adler v1.0.2
  Downloaded anyhow v1.0.65
  Downloaded async-lock v2.5.0
  Downloaded atty v0.2.14
  Downloaded bytes v1.2.1
  Downloaded cfg-if v1.0.0
  Downloaded clap_complete v3.2.5
  Downloaded textwrap v0.15.1
  Downloaded crc32fast v1.3.2
  Downloaded directories v4.0.1
  Downloaded fastrand v1.8.0
  Downloaded event-listener v2.5.3
  Downloaded filetime v0.2.17
  Downloaded flate2 v1.0.24
  Downloaded form_urlencoded v1.0.1
  Downloaded futures-channel v0.3.21
  Downloaded futures-core v0.3.21
  Downloaded futures-executor v0.3.21
  Downloaded futures-io v0.3.21
  Downloaded thiserror v1.0.32
  Downloaded futures-macro v0.3.21
  Downloaded futures-sink v0.3.21
  Downloaded futures-task v0.3.21
  Downloaded futures-lite v1.12.0
  Downloaded time v0.1.44
  Downloaded futures-util v0.3.21
  Downloaded termios v0.3.3
  Downloaded http-body v0.4.5
  Downloaded hex v0.4.3
  Downloaded httpdate v1.0.2
  Downloaded httparse v1.7.1
  Downloaded tinyvec v1.6.0
  Downloaded tinyvec_macros v0.1.0
  Downloaded heck v0.4.0
  Downloaded hashbrown v0.12.3
  Downloaded try-lock v0.2.3
  Downloaded tokio-macros v1.8.0
  Downloaded idna v0.2.3
  Downloaded instant v0.1.12
  Downloaded indexmap v1.9.1
  Downloaded url v2.2.2
  Downloaded version_check v0.9.4
  Downloaded waker-fn v1.1.0
  Downloaded want v0.3.0
  Downloaded byteorder v1.4.3
  Downloaded log v0.4.17
  Downloaded memchr v2.5.0
  Downloaded miniz_oxide v0.5.3
  Downloaded nix v0.24.2
  Downloaded serde_derive v1.0.145
  Downloaded terminal_size v0.1.17
  Downloaded base64 v0.13.0
  Downloaded pager v0.16.1
  Downloaded memoffset v0.6.5
  Downloaded ctor v0.1.23
  Downloaded dirs v4.0.0
  Downloaded clap_lex v0.2.4
  Downloaded errno v0.2.8
  Downloaded futures v0.3.21
  Downloaded quote v1.0.21
  Downloaded ryu v1.0.11
  Downloaded hyper v0.14.20
  Downloaded serde v1.0.145
  Downloaded signal-hook-registry v1.4.0
  Downloaded smallvec v1.9.0
  Downloaded pin-project v1.0.11
  Downloaded termcolor v1.1.3
  Downloaded rustix v0.35.7
  Downloaded syn v1.0.99
  Downloaded unicode-normalization v0.1.21
  Downloaded scopeguard v1.1.0
  Downloaded proc-macro2 v1.0.43
  Downloaded toml v0.5.9
  Downloaded tower-service v0.3.2
  Downloaded linux-raw-sys v0.0.46
  Downloaded xattr v0.2.3
  Downloaded cache-padded v1.2.0
  Downloaded signal-hook v0.1.17
  Downloaded crossbeam-utils v0.8.11
  Downloaded clap_derive v3.2.18
  Downloaded which v4.3.0
  Downloaded value-bag v1.0.0-alpha.9
  Downloaded utf8-width v0.1.6
  Downloaded unicode-width v0.1.10
  Downloaded unicode-ident v1.0.3
  Downloaded unicode-bidi v0.3.8
  Downloaded tracing-core v0.1.29
  Downloaded serde_json v1.0.83
  Downloaded users v0.11.0
  Downloaded unix_socket v0.5.0
  Downloaded termbg v0.4.1
  Downloaded tokio v1.20.1
  Downloaded procfs v0.14.1
  Downloaded tracing v0.1.36
  Downloaded thiserror-impl v1.0.32
  Downloaded tar v0.4.38
  Downloaded libc v0.2.134
  Downloaded pin-utils v0.1.0
  Downloaded crossterm v0.19.0
  Downloaded async-std v1.12.0
  Downloaded cfg-if v0.1.10
  Downloaded atomic-waker v1.0.0
  Downloaded async-channel v1.7.1
  Downloaded strsim v0.10.0
  Downloaded socket2 v0.4.4
  Downloaded async-global-executor v2.2.0
  Downloaded pin-project-internal v1.0.11
  Downloaded os_str_bytes v6.2.0
  Downloaded mio v0.7.14
  Downloaded io-lifetimes v0.7.2
  Downloaded lock_api v0.4.7
  Downloaded hyperlocal v0.8.0
  Downloaded http v0.2.8
  Downloaded itoa v1.0.3
  Downloaded iana-time-zone v0.1.47
  Downloaded either v1.7.0
  Downloaded clap v3.2.22
  Downloaded chrono v0.4.22
  Downloaded concurrent-queue v1.2.4
  Downloaded blocking v1.2.0
  Downloaded async-task v4.3.0
  Downloaded async-io v1.7.0
  Downloaded android_system_properties v0.1.5
  Downloaded getch v0.3.1
  Downloaded dockworker v0.0.24
  Downloaded autocfg v1.1.0
  Downloaded slab v0.4.7
  Downloaded proc-macro-error v1.0.4
  Downloaded polling v2.2.0
  Downloaded pin-project-lite v0.2.9
  Downloaded percent-encoding v2.1.0
  Downloaded parking_lot_core v0.9.3
  Downloaded parking_lot_core v0.8.5
  Downloaded parking_lot v0.12.1
  Downloaded parking_lot v0.11.2
  Downloaded num_cpus v1.13.1
  Downloaded num-traits v0.2.15
  Downloaded num-integer v0.1.45
   Compiling libc v0.2.134
   Compiling proc-macro2 v1.0.43
   Compiling autocfg v1.1.0
   Compiling unicode-ident v1.0.3
   Compiling quote v1.0.21
   Compiling syn v1.0.99
   Compiling cfg-if v1.0.0
   Compiling version_check v0.9.4
   Compiling once_cell v1.15.0
   Compiling log v0.4.17
   Compiling futures-core v0.3.21
   Compiling pin-project-lite v0.2.9
   Compiling memchr v2.5.0
   Compiling futures-io v0.3.21
   Compiling smallvec v1.9.0
   Compiling scopeguard v1.1.0
   Compiling futures-channel v0.3.21
   Compiling pin-utils v0.1.0
   Compiling bitflags v1.3.2
   Compiling futures-task v0.3.21
   Compiling cache-padded v1.2.0
   Compiling parking v2.0.0
   Compiling fastrand v1.8.0
   Compiling parking_lot_core v0.9.3
   Compiling waker-fn v1.1.0
   Compiling futures-util v0.3.21
   Compiling futures-sink v0.3.21
   Compiling bytes v1.2.1
   Compiling serde_derive v1.0.145
   Compiling itoa v1.0.3
   Compiling event-listener v2.5.3
   Compiling httparse v1.7.1
   Compiling async-task v4.3.0
   Compiling fnv v1.0.7
   Compiling parking_lot_core v0.8.5
   Compiling tinyvec_macros v0.1.0
   Compiling serde v1.0.145
   Compiling io-lifetimes v0.7.2
   Compiling crc32fast v1.3.2
   Compiling lazy_static v1.4.0
   Compiling crossbeam-utils v0.8.11
   Compiling atomic-waker v1.0.0
   Compiling matches v0.1.9
   Compiling try-lock v0.2.3
   Compiling heck v0.4.0
   Compiling percent-encoding v2.1.0
   Compiling httpdate v1.0.2
   Compiling os_str_bytes v6.2.0
   Compiling hashbrown v0.12.3
   Compiling hex v0.4.3
   Compiling rustix v0.35.7
   Compiling adler v1.0.2
   Compiling serde_json v1.0.83
   Compiling tower-service v0.3.2
   Compiling unicode-bidi v0.3.8
   Compiling termcolor v1.1.3
   Compiling strsim v0.10.0
   Compiling procfs v0.14.1
   Compiling byteorder v1.4.3
   Compiling ryu v1.0.11
   Compiling anyhow v1.0.65
   Compiling textwrap v0.15.1
   Compiling cfg-if v0.1.10
   Compiling mime v0.3.16
   Compiling unicode-width v0.1.10
   Compiling either v1.7.0
   Compiling utf8-width v0.1.6
   Compiling base64 v0.13.0
   Compiling slab v0.4.7
   Compiling lock_api v0.4.7
   Compiling num-traits v0.2.15
   Compiling num-integer v0.1.45
   Compiling tokio v1.20.1
   Compiling memoffset v0.6.5
   Compiling indexmap v1.9.1
   Compiling instant v0.1.12
   Compiling tracing-core v0.1.29
   Compiling value-bag v1.0.0-alpha.9
   Compiling proc-macro-error-attr v1.0.4
   Compiling proc-macro-error v1.0.4
   Compiling concurrent-queue v1.2.4
   Compiling async-lock v2.5.0
   Compiling http v0.2.8
   Compiling tinyvec v1.6.0
   Compiling form_urlencoded v1.0.1
   Compiling clap_lex v0.2.4
   Compiling miniz_oxide v0.5.3
   Compiling byte-unit v4.0.14
   Compiling tracing v0.1.36
   Compiling http-body v0.4.5
   Compiling unicode-normalization v0.1.21
   Compiling socket2 v0.4.4
   Compiling num_cpus v1.13.1
   Compiling signal-hook-registry v1.4.0
   Compiling android_system_properties v0.1.5
   Compiling errno v0.2.8
   Compiling time v0.1.44
   Compiling dirs-sys v0.3.7
   Compiling filetime v0.2.17
   Compiling xattr v0.2.3
   Compiling atty v0.2.14
   Compiling unix_socket v0.5.0
   Compiling terminal_size v0.1.17
   Compiling termios v0.3.3
   Compiling which v4.3.0
   Compiling async-channel v1.7.1
   Compiling futures-lite v1.12.0
   Compiling flate2 v1.0.24
   Compiling idna v0.2.3
   Compiling iana-time-zone v0.1.47
error[E0308]: arguments to this function are incorrect
    --> /home/builder/.cargo/registry/src/github.com-1ecc6299db9ec823/rustix-0.35.7/src/backend/libc/fs/makedev.rs:15:14
     |
15   |     unsafe { c::makedev(maj as i32, min as i32) }
     |              ^^^^^^^^^^ ----------  ---------- expected `u32`, found `i32`
     |                         |
     |                         expected `u32`, found `i32`
     |
note: function defined here
    --> /home/builder/.cargo/registry/src/github.com-1ecc6299db9ec823/libc-0.2.134/src/unix/linux_like/android/mod.rs:2794:20
     |
2794 |     pub {const} fn makedev(ma: ::c_uint, mi: ::c_uint) -> ::dev_t {
     |                    ^^^^^^^
help: you can convert an `i32` to a `u32` and panic if the converted value doesn't fit
     |
15   |     unsafe { c::makedev((maj as i32).try_into().unwrap(), min as i32) }
     |                         +          +++++++++++++++++++++
help: you can convert an `i32` to a `u32` and panic if the converted value doesn't fit
     |
15   |     unsafe { c::makedev(maj as i32, (min as i32).try_into().unwrap()) }
     |                                     +          +++++++++++++++++++++

For more information about this error, try `rustc --explain E0308`.
error: could not compile `rustix` due to previous error
warning: build failed, waiting for other jobs to finish...
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2022-10-05 06:29:24 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/3187674103">3187674103</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/procs/0001-rustix-upstream-fix-libc-removing-unsafe-on-makedev.diff
+++ b/packages/procs/0001-rustix-upstream-fix-libc-removing-unsafe-on-makedev.diff
@@ -1,0 +1,71 @@
+From d96e9957e5725bb67e8ba617cb8bfe087a271ae0 Mon Sep 17 00:00:00 2001
+From: ETKNeil <89385146+ETKNeil@users.noreply.github.com>
+Date: Tue, 20 Sep 2022 13:39:18 +0200
+Subject: [PATCH] Fix libc removing unsafe on makedev
+
+---
+ Cargo.toml                     | 6 +++---
+ src/backend/libc/fs/makedev.rs | 6 +++---
+ 2 files changed, 6 insertions(+), 6 deletions(-)
+
+# diff --git a/Cargo.toml b/Cargo.toml
+# index d9735cb8..dcde3702 100644
+# --- a/Cargo.toml
+# +++ b/Cargo.toml
+# @@ -40,7 +40,7 @@ once_cell = { version = "1.5.2", optional = true }
+#  [target.'cfg(all(not(rustix_use_libc), not(miri), target_os = "linux", any(target_arch = "x86", all(target_arch = "x86_64", target_pointer_width = "64"), all(target_endian = "little", any(target_arch = "arm", all(target_arch = "aarch64", target_pointer_width = "64"), target_arch = "powerpc64", target_arch = "riscv64", target_arch = "mips", target_arch = "mips64")))))'.dependencies]
+#  linux-raw-sys = { version = "0.0.46", default-features = false, features = ["general", "errno", "ioctl", "no_std"] }
+#  libc_errno = { package = "errno", version = "0.2.8", default-features = false, optional = true }
+# -libc = { version = "0.2.126", features = ["extra_traits"], optional = true }
+# +libc = { version = "0.2.133", features = ["extra_traits"], optional = true }
+ 
+#  # Dependencies for platforms where only libc is supported:
+#  #
+# @@ -48,7 +48,7 @@ libc = { version = "0.2.126", features = ["extra_traits"], optional = true }
+#  # backend, so enable its dependencies unconditionally.
+#  [target.'cfg(any(rustix_use_libc, miri, not(all(target_os = "linux", any(target_arch = "x86", all(target_arch = "x86_64", target_pointer_width = "64"), all(target_endian = "little", any(target_arch = "arm", all(target_arch = "aarch64", target_pointer_width = "64"), target_arch = "powerpc64", target_arch = "riscv64", target_arch = "mips", target_arch = "mips64")))))))'.dependencies]
+#  libc_errno = { package = "errno", version = "0.2.8", default-features = false }
+# -libc = { version = "0.2.126", features = ["extra_traits"] }
+# +libc = { version = "0.2.133", features = ["extra_traits"] }
+ 
+#  # Additional dependencies for Linux with the libc backend:
+#  #
+# @@ -69,7 +69,7 @@ features = [
+ 
+#  [dev-dependencies]
+#  tempfile = "3.2.0"
+# -libc = "0.2.126"
+# +libc = "0.2.133"
+#  libc_errno = { package = "errno", version = "0.2.8", default-features = false }
+#  io-lifetimes = { version = "1.0.0-rc1", default-features = false }
+#  # Don't upgrade to serial_test 0.7 for now because it depends on a
+diff --git a/src/backend/libc/fs/makedev.rs b/src/backend/libc/fs/makedev.rs
+index d9089e7f..08ecd872 100644
+--- a/src/backend/libc/fs/makedev.rs
++++ b/src/backend/libc/fs/makedev.rs
+@@ -5,14 +5,14 @@ use crate::fs::Dev;
+ #[cfg(not(any(target_os = "android", target_os = "emscripten")))]
+ #[inline]
+ pub(crate) fn makedev(maj: u32, min: u32) -> Dev {
+-    unsafe { c::makedev(maj, min) }
++    c::makedev(maj, min)
+ }
+ 
+ #[cfg(all(target_os = "android", not(target_pointer_width = "32")))]
+ #[inline]
+ pub(crate) fn makedev(maj: u32, min: u32) -> Dev {
+     // Android's `makedev` oddly has signed argument types.
+-    unsafe { c::makedev(maj as i32, min as i32) }
++    c::makedev(maj, min)
+ }
+ 
+ #[cfg(all(target_os = "android", target_pointer_width = "32"))]
+@@ -30,7 +30,7 @@ pub(crate) fn makedev(maj: u32, min: u32) -> Dev {
+ #[inline]
+ pub(crate) fn makedev(maj: u32, min: u32) -> Dev {
+     // Emscripten's `makedev` has a 32-bit return value.
+-    Dev::from(unsafe { c::makedev(maj, min) })
++    Dev::from(c::makedev(maj, min))
+ }
+ 
+ #[cfg(not(any(target_os = "android", target_os = "emscripten")))]

--- a/packages/procs/build.sh
+++ b/packages/procs/build.sh
@@ -2,11 +2,29 @@ TERMUX_PKG_HOMEPAGE=https://github.com/dalance/procs
 TERMUX_PKG_DESCRIPTION="A modern replacement for ps"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.13.1"
+TERMUX_PKG_VERSION="0.13.2"
 TERMUX_PKG_SRCURL=https://github.com/dalance/procs/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=29a5058f8389986e5b17117d3ee4f78a2bf9c8886e07b16a49a94e5fc59a6850
+TERMUX_PKG_SHA256=521b25cbb3fac8b6f3cfd33c4d58f269e7d6637a89f6b932f144a8bed7914ec0
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_AUTO_UPDATE=true
+
+termux_step_pre_configure() {
+	termux_setup_rust
+
+	export CFLAGS="${TARGET_CFLAGS}"
+
+	: "${CARGO_HOME:=$HOME/.cargo}"
+	export CARGO_HOME
+
+	rm -rf $CARGO_HOME/registry/src/github.com-*/libc-*
+	rm -rf $CARGO_HOME/registry/src/github.com-*/rustix-*
+	cargo fetch --target "${CARGO_TARGET_NAME}"
+
+	for d in $CARGO_HOME/registry/src/github.com-*/rustix-*; do
+		patch --silent -p1 -d ${d} < $TERMUX_PKG_BUILDER_DIR/0001-rustix-upstream-fix-libc-removing-unsafe-on-makedev.diff
+	done
+
+}
 
 # This package contains makefiles to run the tests. So, we need to override build steps.
 termux_step_make() {

